deployer/scripts/util.sh

```diff
@@ -546,3 +546,23 @@ function wait_for_fluentd_to_catch_up() {
     echo END wait_for_fluentd_to_catch_up at `date -u --rfc-3339=ns`
     return $rc
 }
+
+docker_uses_journal() {
+    # note the unintuitive logic - in this case, a 0 return means true, and a 1
+    # return means false
+    # need to be able to handle cases like
+    # OPTIONS='--log-driver=json-file ....' # or use --log-driver=journald
+    # if "log-driver" is set in /etc/docker/daemon.json, assume that it is
+    # authoritative
```

Comment:
@stevekuznetsov it does return false - it should fall through to line 567 and `return 1` - false